app/assets/javascripts/hideShowPassword.js

```diff
@@ -0,0 +1,453 @@
+(function (factory) {
```

Comment:
I guess I over thought it. I did like the ability to add a "hide/show" button
inside the field on focus, but it would have been difficult to work with given
our giant buttons and having to override a bunch of stuff.

Seems that was too simple. I'll take a crack at it when I get back home from
dinner.



- Jared On Thu, Nov 12, 2015 at 5:28 PM, Alex Brinkman < notifications@github.com [notifications@github.com] > wrote:
In app/assets/javascripts/hideShowPassword.js
[https://github.com/PublicLibraryAssoc/cpldl/pull/33#discussion_r44736161] :

> @@ -0,0 +1,453 @@
> +(function (factory) {


Do we really need all of this? It seems that we could flip the type attribute on
the password input in the same method js method that you're catching the
checkbox event. Is it not as simple as that?

—
Reply to this email directly or view it on GitHub
[https://github.com/PublicLibraryAssoc/cpldl/pull/33/files#r44736161] .[https://github.com/notifications/beacon/AA_S9xvjtuPwK-_JS8Eg9vUBDf3E_8iIks5pFSYzgaJpZM4Ghce_.gif]